src/clang_tu.cc

```diff
@@ -45,7 +45,8 @@ static Pos Decomposed2LineAndCol(const SourceManager &SM,
                                  std::pair<FileID, unsigned> I) {
   int l = SM.getLineNumber(I.first, I.second) - 1,
       c = SM.getColumnNumber(I.first, I.second) - 1;
-  return {(int16_t)std::min(l, INT16_MAX), (int16_t)std::min(c, INT16_MAX)};
+  return {(int16_t)std::min<int16_t>(l, INT16_MAX),
```

Comment:
I am a bit surprised. Doesn't `(int16_t)std::min(l, INT16_MAX)` compile on MSVC 2017?

`std::min<int16_t>` narrow converts `l` to `int16_t`, which is not intended.

Reply:
It won't be compiled on VC2017, since the compiler failed to infer the parameter type by return type.